ui/app/clinical/consultation/controllers/conceptSetPageController.js

```diff
@@ -240,11 +241,36 @@ angular.module('bahmni.clinical')
                         });
                         if (currentLabel) { label = currentLabel.display; }
                     }
-                    forms.push(new Bahmni.ObservationForm(formUuid, $rootScope.currentUser,
-                        formName, formVersion, observations, label));
+                    if ($scope.isFormEditableByTheUser(observationForm)) {
+                        var tempForm = new Bahmni.ObservationForm(formUuid, $rootScope.currentUser,
```

Comment:
@buvaneswari-arun can you please change _tempForm_ to something more meaningful

Reply:
changed into new form